src/Errors/AbstractError.ts

```diff
@@ -5,6 +5,7 @@ abstract class AbstractError extends Error {
 
 	constructor( message:string ) {
 		super( message );
+		Error.captureStackTrace( this, this.constructor );
```

Comment:
Why are you adding this line? I can't find documentation about it except for the special node Error object. Are you sure it exists in the browser?